src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,12 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    transition: color 300ms linear, transform 300ms linear;
```

Comment:
Great job! There is one recommendation for your code) Maybe you can create a variable with transition time because in all of the cases, we use the same time)

Reply:
Fixed ✔